maintainers/maintainer-list.nix

```diff
@@ -20711,6 +20711,12 @@
     githubId = 18656090;
     name = "Yuki Takagi";
   };
+  takeda = {
+    name = "Derek Kuliński";
+    email = "d@kulinski.us";
+    github = "takeda";
+    githubId = 411978;
+  };
```

Comment:
This should be a seperate commit: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions